test/functional/api/cas/installer.py

```diff
@@ -23,21 +24,21 @@ def install_opencas():
         "./configure && "
         "make -j")
     if output.exit_code != 0:
-        TestRun.exception(
+        raise CmdException(
             f"Make command executed with nonzero status: {output.stdout}\n{output.stderr}")
 
     TestRun.LOGGER.info("Installing Open CAS")
     output = TestRun.executor.run(
         f"cd {TestRun.plugins['opencas'].working_dir} && "
         f"make install")
     if output.exit_code != 0:
-        TestRun.exception(
+        raise CmdException(
             f"Error while installing Open CAS: {output.stdout}\n{output.stderr}")
```

Comment:
same here and in the following CmdExceptions

Reply:
Done.